src/main/kotlin/biokotlin/util/MergeGVCFUtil.kt

```diff
@@ -0,0 +1,361 @@
+package biokotlin.util
+
+import biokotlin.genome.Position
+import htsjdk.variant.variantcontext.Allele
+import htsjdk.variant.variantcontext.GenotypeBuilder
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.variantcontext.VariantContextBuilder
+import htsjdk.variant.variantcontext.writer.Options
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder
+import htsjdk.variant.vcf.VCFHeader
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.Channel
+import java.io.File
+import kotlin.system.measureNanoTime
+
+fun mergeGVCFs(inputDir: String, outputFile: String) {
```

Comment:
Should this file be named MergeGVCFUtils?

Reply:
Ok, I changed that